Lesson_02/fibonacciOne/fibonacciOne.go

```diff
@@ -0,0 +1,8 @@
+package fibonacciOne
+
+func Fib(n int) int {
```

Comment:
Recursion is not a best way to do at Go. Check that out https://stackoverflow.com/questions/12102675/tail-call-optimization-in-go
Or google about `tail recursion in go`

Reply:
But that works=)